pkg/watch/health.go

```diff
@@ -0,0 +1,257 @@
+package watch
+
+import (
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/square/p2/pkg/health"
+	"github.com/square/p2/pkg/kp"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/preparer"
```

Comment:
it's a little strange to import the preparer here, but I realize that's an effect of moving the config stuff out of `bin/preparer`. Fine for now but I think we can do this better with a little reordering.